### PR TITLE
Set English as Jitt's default display language. fixes #1

### LINKED ISSUE #1: When LC is not English JITT should maintain English in some places
1. Original string shows up translated if we have a translation. Original string should always be English
2. Select languages list is half locale codes, half in the language. Should be always English

### PR DESCRIPTION
**Pull request by amirnissim**

### CODE REVIEW

File: src/main/java/me/everything/jittlib/Jitt.java [hunk 1]
```diff
@@ -370,6 +377,15 @@ private void getDataFromServer() {
         }
     }
 
+    public int getResourceIdForString(String string) {
+        Entry entry = mResourcesEntries.get(string);
+        return entry.resId;
```
Comment (akariv) [inline src/main/java/me/everything/jittlib/Jitt.java]: can this be null?

File: src/main/java/me/everything/jittlib/JittTranslateActivity.java [hunk 2]
```diff
@@ -78,6 +81,7 @@ public void onPostAction() {
 
     private List<Object> mItemsList;
 
+    @TargetApi(Build.VERSION_CODES.JELLY_BEAN_MR1)
```
Comment (akariv) [inline src/main/java/me/everything/jittlib/JittTranslateActivity.java]: why did you add this?

File: src/main/java/me/everything/jittlib/Jitt.java [hunk 3]
```diff
@@ -109,8 +116,8 @@ public void run() {
                 lang = "he";
             }
             if (!mLocaleList.containsKey(lang)) {
-                mLocaleList.put(lang, l.getDisplayLanguage());
-                localeCountries.add(l.getLanguage());
+                mLocaleList.put(lang, l.getDisplayLanguage(mDisplayLocale));
```
Comment (akariv) [inline src/main/java/me/everything/jittlib/Jitt.java]: Why this change?

Comment (amirnissim) [inline src/main/java/me/everything/jittlib/Jitt.java]: Martha requests locales will always display in English

File: src/main/java/me/everything/jittlib/JittTranslateActivity.java (hunk 2 above)
Comment (amirnissim) [inline src/main/java/me/everything/jittlib/JittTranslateActivity.java]: `createConfigurationContext` requires API level 17

File: src/main/java/me/everything/jittlib/Jitt.java (hunk 1 above)
Comment (amirnissim) [inline src/main/java/me/everything/jittlib/Jitt.java]: yes. you're right. will handle null.

Comment (amirnissim) [inline src/main/java/me/everything/jittlib/Jitt.java]: scratch that. it's always an `int`

Comment (akariv) [inline src/main/java/me/everything/jittlib/Jitt.java]: I meant what if entry == null

File: src/main/java/me/everything/jittlib/JittTranslateActivity.java (hunk 2 above)
Comment (akariv) [inline src/main/java/me/everything/jittlib/JittTranslateActivity.java]: what if API level < 17?

File: src/main/java/me/everything/jittlib/JittTranslateActivity.java [hunk 4]
```diff
@@ -90,8 +93,18 @@ protected void onCreate(Bundle savedInstanceState) {
         getSupportActionBar().setDisplayHomeAsUpEnabled(true);
         getSupportActionBar().setBackgroundDrawable(new ColorDrawable(Color.parseColor("#13b476")));
 
-        TextView originalString = (TextView)findViewById(R.id.original_string);
-        originalString.setText(mString);
+        // display the original string in Jitt's locale if possible
+        // otherwise use device locale
+        String originalString = mString;
+        int resId = Jitt.getInstance().getResourceIdForString(mString);
+        if (Build.VERSION.SDK_INT > Build.VERSION_CODES.JELLY_BEAN_MR1 && resId > 0) {
```
Comment (akariv) [inline src/main/java/me/everything/jittlib/JittTranslateActivity.java]: > or >= ?

Comment (amirnissim) [inline src/main/java/me/everything/jittlib/JittTranslateActivity.java]: fixed. 10x.
